sceptre/config/graph.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+
+"""
+sceptre.config.graph
+This module implements a StackConfig graph, which stores a directed graph
+of a stack's dependencies.
+"""
+
+import logging
+import networkx as nx
+from sceptre.exceptions import CircularDependenciesError
+
+
+class StackDependencyGraph(object):
+    """
+    A Directed Graph representing the relationship between Stack config
+    dependencies. Responsible for initalizing the graph object based on
+    a given inital stack config path.
+
+    :param inital_config_path: The inital stack config you want to generate
```

Comment:
This parameter documentation is out of date, it no longer takes a string `initial_config_path`